src/modules/75-cf/components/SideNav/SideNav.tsx

```diff
@@ -90,21 +90,25 @@ export default function CFSideNav(): React.ReactElement {
                 to={routes.toAccessControl({ ...params, module: 'cf' })}
                 label={getString('accessControl')}
               />
-              {FF_GITSYNC && !isGitSimplificationEnabled && (
-                <>
-                  <SidebarLink
-                    label={getString('connectorsLabel')}
-                    to={routes.toConnectors({ ...params, module: 'cf' })}
-                  />
-                  <SidebarLink label={getString('common.secrets')} to={routes.toSecrets({ ...params, module: 'cf' })} />
-                  {!FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && (
+              {(FF_GITSYNC && !isGitSimplificationEnabled) ||
+                (FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && isGitSimplificationEnabled && (
```

Comment:
Looks like your parens are off
```suggestion
              {((FF_GITSYNC && !isGitSimplificationEnabled) ||
                (FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && isGitSimplificationEnabled)) && (
```

Reply:
The linter drops them

Reply:
Nope I'm an idiot, fixed